css/style.css

```diff
@@ -0,0 +1,360 @@
+/* Style.css */
+* {
+  box-sizing: border-box;
+}
+
+.clearfix:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+body {
+  font-family: helvetica;
+  font-size: 16px;
+}
+
+.stroke {
+  border: 3px solid #fff;
+  background: rgba(0, 0, 0, 0.75);
+  padding: 15px;
+  box-shadow: 5px 5px 5px rgba(0, 0, 0, 0.5);
+}
+
+h1 {
+  font-size:50px;
+  font-family: helvetica;
+  font-weight: bold;
+}
+
+.bold {
+  font-weight: bold;
+}
+
+.align-left {
+  float: left;
+}
+
+.align-right {
+  float: right;
+}
+
+#container {
+  width: 100%;
+  margin: auto;
+  height: 500px;
+}
+
+.hero {
+  text-align: center;
+  background-size: cover;
+  height: 100vh;
+  background: url(../images/art-main.png) no-repeat center top;
+    background-size: cover;
+    background-attachment: fixed;
+    background-position: center;
+    filter: grayscale(100%);
```

Comment:
Also super cool use of filter here rather than altering your image - gives you more flexibility in the long run.